Library/Control/ControllerInterface.hh

```diff
@@ -0,0 +1,61 @@
+/*
+ * ControllerInterface.hh
+ *
+ *  Created on: Jan 24, 2022
+ *      Author: N.S. Oblath
+ */
+
+#ifndef NYMPH_CONTROLLERINTERFACE_HH_
+#define NYMPH_CONTROLLERINTERFACE_HH_
+
+#include <exception>
+#include <string>
+
+namespace Nymph
+{
+
+    /*!
```

Comment:
Could you explain how this class is used in the current structure? I see that it's used for defining both the `Control` and `ControlAccess` classes, but it also looks like you're redefining all of the functions present within `ControllerInterface` anyway. Is this class being used just to make sure that those two other classes are initialized identically? Or maybe I'm misunderstanding how this class works with the others, or it's being used in more ways than just by these two classes.

Reply:
I thought it would be useful to have a common interface definition for Controllers and `ControlAccess`, even though at the time I didn't have a specific use of that common interface.  As it has turned out, that's not the case.  And it turns out to be problematic in a branch that I'm working on now.  So I'll leave it in here, but it's going to be removed soon.